### ISSUE TITLE
main driver 4 - year of species discovery

### ISSUE THREAD

**Issue by tmesaglio** [tmesaglio/Australian-plant-photos#21]
Visualising when each unphotographed species was originally described. I fixed up the code so that each year is the actual original date, including for entities that were comb. nov., nom. nov., etc (so basically I found all the basionym dates for things that moved genus, etc)

![image](https://user-images.githubusercontent.com/48739968/165468527-8d131bb0-2e3e-4767-9b93-75021346159b.png)

**Comment by tmesaglio** [comment]
most of the stuff from the 1700s is species with distributions also outside Australia

**Comment by wcornwell** [comment]
> most of the stuff from the 1700s is species with distributions also outside Australia

Yeah I noticed this too. Interesting historical tidbit to fit into some future project
